extra/utils/validation.js

```diff
@@ -0,0 +1,40 @@
+let c=0,d=0;
+const users=[ {
+traineeEmail : "trainee1@successive.tech",
+reviewerEmail : "reviewer1@successive.tech"},
+
+{
+    traineeEmail : "trainee2@successive.tech",
+    reviewerEmail : "reviewer2@successive.tech"},
+
+{
+    traineeEmail : "trainee3@successive.tech",
+    reviewerEmail : "reviewer3@successive.tech"},
+
+{
+traineeEmail : "trainee4@successive.tech",
+reviewerEmail : "reviewer4@successive.tech"},
+];   
```

Comment:
indent the code properly

Reply:
Done

Reply:
need proper indentation

Reply:
[Indentation Guidelines](https://github.com/airbnb/javascript )- use this for reference